### ISSUE TITLE
Adding a price filter

### ISSUE THREAD

**Issue by greenbrook** [StarbeamOne/home#89]
We've been already discussing that "latest" and "recent" filters are quite similar. What if to give the collectors the ability to sort by price and be able to purchase more with a lower budget?

![Frame 46](https://user-images.githubusercontent.com/125400168/227795075-2cbd9517-376e-45b7-94ee-a899cfbe1a2c.png)

**Comment by Mike-E-angelo** [comment]
We could do this with a grid easily.  Tiles are their own animal, and will take some work.  Right now, we have the tiles ordered by created date descending.

We'd have to have some way of saying:

Order by:
- Date Descending (default/current experience)
- Date Ascending
- Price Ascending
- Price Descending

**Comment by greenbrook** [comment]
Is JavaScript capable of it?

**Comment by Mike-E-angelo** [comment]
Yes but so is .NET :)  For reference, everything you see w/ starbeam.one runs in all modern browsers and consists of ~135,000 lines of C#, and about 150 lines of custom JavaScript.  Very little JavaScript is actually used/found in our solution.  There is other JavaScript used but it is provided/managed by control vendors used by our solution.

**Comment by Mike-E-angelo** [comment]
you can see an example of price sorting ascending/descending in the Transaction History grid:

![image](https://user-images.githubusercontent.com/2931376/227801767-3d3ed901-3f55-4331-8648-1a87787cb136.png)

https://alpha.starbeam.one/profile/transactions
